examples/dlint/lint.ts

```diff
@@ -0,0 +1,3 @@
+import * as xD from "xD";
```

Comment:
This was for initial testing - can remove this now :)

Reply:
thanks, I've removed 👍